app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -20,6 +20,31 @@ class Api::V2::ContentViewsController < Api::V2::ApiController
       param :component_ids, Array, :desc => N_("List of component content view version ids for composite views")
     end
 
+    api :POST, "/content_views/:id/export", N_("Export a content view to disk")
+    param :id, :identifier, :desc => N_("Content view identifier"), :required => true
```

Comment:
I think I'm going to change it to CVV with date-based incremental per your original comment, and we can revisit it once https://pulp.plan.io/issues/1543 is picked up at a later date.